packages/web/components/Technology/Details/Section.js

```diff
@@ -13,6 +15,12 @@ const Section = ({ title, color, hideWhenIsEmpty, children = [] }) => {
 		<Container>
 			<Title color={color}>
 				<h4>{title}</h4>
+				{rating && (
+					<RatingText aria-label="Média de Avaliações">
+						Avaliação dessa tecnologia: <strong>{rating}</strong>{' '}
+						<Rating value={rating} size={3} readonly />
+					</RatingText>
+				)}
```

Comment:
It seems to me that we will only use the `rating` prop in this single case. What do you think about changing this to a `render prop`?